### PR TITLE
New Files Added: Assignment9

### PR DESCRIPTION
**Pull request by vinod2406**

### CODE REVIEW

**Review by akash19972306 — APPROVED** [review]

File: vinod/html/ASSIN9/Boostrap.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+
+        <!-- Bootstrap CSS -->
```
Comment (sourabh513) [inline vinod/html/ASSIN9/Boostrap.html]: remove commend lines

File: vinod/html/ASSIN9/Boostrap.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        <link
+            rel="stylesheet"
+            href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+            integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+            crossorigin="anonymous"
+        />
+
```
Comment (sourabh513) [inline vinod/html/ASSIN9/Boostrap.html]: remove extra line

File: vinod/html/ASSIN9/Boostrap.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        <title>Hello, world!</title>
+        <style>
+            /*.block {
```
Comment (sourabh513) [inline vinod/html/ASSIN9/Boostrap.html]: remove comment line

File: vinod/html/ASSIN9/Boostrap.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    left: 26px;
+    height: 33px;
+    width: 120px;
+    background-color: purple;
+    color: white;
+} */
+        </style>
+    </head>
+    <body>
+        <!-- <div class="consultantzone">
```
Comment (sourabh513) [inline vinod/html/ASSIN9/Boostrap.html]: remove comment line

File: vinod/html/ASSIN9/assin9.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>TimesJob.com</title>
+        <link rel="stylesheet" href="ass9.style.css" />
+        <link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaina+2&display=swap" rel="stylesheet" />
+
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: remove extra line

File: vinod/html/ASSIN9/assin9.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+                    </ul>
+            </div>
+        </header>
+            <header>
+                <div class="block2">
+                    <div>      
+                        <img src="timesjob.png" alt="timesjob" class="img">         
+                   </div>
+                    <div class="block4">
+                            <button class="btn">Login</button>
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: wrong indentation

File: vinod/html/ASSIN9/assin9.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+                            <button class="btn2">Register</button>
+                    </div>
+                    <div>
+                        <img src="23.jpg" alt="notification" class="img3"></li>  
+                    </div>
+                </div>
+            </header>
+                    <div class="r">
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: wrong identtion

File: vinod/html/ASSIN9/assin9.html [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+                                <span class="upload-icon"></span>
+                                <h5 class="h1">NEW TO TIMES JOBS</h5>
+                                <div>
+                                    <img src="upload.webp" alt="upload" class="uploadicon">
+                                </div>
+                                <div>
+                                    <button class="uploadbtn">UPLOAD YOUR RESUME</button>
+
+                                </div>
+
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: remove extra lines

File: vinod/html/ASSIN9/assin9.html [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+                                </div>
+                                
+
+                    </div>
+                    <div>
+                        <img src="24.png" alt="background image" class="img2">
+                    </div>
+                    <div>
+                        <img src="25.gif" alt="background image" class="img4">
+
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: remove extra line

File: vinod/html/ASSIN9/assin9.html [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+                    </div>
+                    <div>
+                        <img src="24.png" alt="background image" class="img2">
+                    </div>
+                    <div>
+                        <img src="25.gif" alt="background image" class="img4">
+                        
+                    </div>
+
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: remove extra line

File: vinod/html/ASSIN9/assin9.html [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+                                    <h5>Sales $ Marketing Jobs</h5>
+                                    <h5 class="z10"><span style="color: red">32589 Jobs</span></h5>
+                                </div>
+                                <div class="z11">
+                                    <h5 class="z13">All Jobs</h5>
+                                    <h5 class="z12"><span style="color: red">246445 Jobs</span></h5>
+                                </div>
+                            </div>
+                        </section>
+                        <!-- <div class="consultantzone">
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: remove command lines

File: vinod/html/ASSIN9/assin9.html [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+                            <h5 class="delhi">DELHI</h5>
+                            <h5 class="ahemdabad">AHEMDABAD</h5>
+                            <button class="viewbtn">VIEW NATIONAL</button>
+                            </div> -->
+                            
+                        <div>
+                            <img src="g.png" alt="bestconsultant" class="img6">
+                            
+                    </div>
+
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: remove extra lines

File: vinod/html/ASSIN9/assin9.html [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+                    
+                    
+                
+    </body>
+</html>
```
Comment (sourabh513) [inline vinod/html/ASSIN9/assin9.html]: add new line at the end of the program

File: vinod/html/ASSIN7/presonal.html [hunk 14]
```diff
@@ -55,4 +55,4 @@ <h1>Academic details</h1>
         </form>
       </block>
     </body>
```
Comment (Pawan1005) [inline vinod/html/ASSIN7/presonal.html:57]: make correct indentation

File: vinod/html/ASSIN9/assin9.html [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+                        <li><img src="outline_tty_black_24dp.png" alt="call"></li>
+                        <li>  888888888</li>                   
+                        <li>live chat(10am to 5pm)</li>
+                        <li><a href="#" style="color: blue;">Recruters login</a></li>
+                        <li><a href="#" style="color: blue;">Buy Online</a></li>
+                    </ul>
+               </div>
+        </header>
+        <header>
+                <div class="block2">
```
Comment (Pawan1005) [inline vinod/html/ASSIN9/assin9.html:35]: make correct indentation

File: vinod/html/ASSIN9/assin9.html [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+                    </ul>
+                <div class="tut2">
+                    <ul>
+                        <li><img src="outline_tty_black_24dp.png" alt="call"></li>
+                        <li>  888888888</li>                   
+                        <li>live chat(10am to 5pm)</li>
+                        <li><a href="#" style="color: blue;">Recruters login</a></li>
+                        <li><a href="#" style="color: blue;">Buy Online</a></li>
+                    </ul>
+               </div>
```
Comment (Pawan1005) [inline vinod/html/ASSIN9/assin9.html]: make correct indentation

File: vinod/html/ASSIN9/assin9.html [hunk 17]
```diff
@@ -0,0 +1,112 @@
...
+                                    <h5 class="z13">All Jobs</h5>
+                                    <h5 class="z12"><span style="color: red">246445 Jobs</span></h5>
+                                </div>
+                            </div>
+                    </section>
+                        <div>
+                            <img src="g.png" alt="bestconsultant" class="img6">
+                        </div>
+     </body>
+</html>
```
Comment (Pawan1005) [inline vinod/html/ASSIN9/assin9.html:110]: make correct indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by sourabh513 — APPROVED** [review]

**Pull request merged**